bundles/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/internal/protocol/CommonCommands.java

```diff
@@ -0,0 +1,460 @@
+                        var broadcast = new InetAddress[3];
+                        broadcast[0] = InetAddress.getByName("224.0.0.1");
+                        broadcast[1] = InetAddress.getByName("255.255.255.255");
+                        broadcast[2] = interfaceAddress.getBroadcast();
+                        for (InetAddress bc : broadcast) {
+                            // Send the broadcast package!
+                            if (bc != null) {
+                                try {
+                                    ArrayList<Byte> buf = buildVNetFrame(macacoFrame, "255.255.255.255", iUserIndex,
+                                            iNodeIndex);
+                                    byte[] merd = toByteArray(buf);
+                                    var packet = new DatagramPacket(merd, merd.length, bc,
+                                            SoulissUDPConstants.SOULISS_GATEWAY_DEFAULT_PORT);
+                                    // Datagramsocket creation
+                                    var channel = DatagramChannel.open();
+                                    sender = channel.socket();
+                                    sender.setReuseAddress(true);
+                                    sender.setBroadcast(true);
+
+                                    var sa = new InetSocketAddress(230);
+                                    sender.bind(sa);
+
+                                    sender.send(packet);
+                                    logger.debug("Request packet sent to: {} Interface: {}", bc.getHostAddress(),
+                                            networkInterface.getDisplayName());
+
+                                } catch (IOException e) {
+                                    logger.debug("IO error: {}", e.getMessage());
+                                } catch (Exception e) {
+                                    logger.debug("{}", e.getMessage(), e);
+                                } finally {
+                                    if ((sender != null) && (!sender.isClosed())) {
+                                        sender.close();
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        } catch (SocketException | UnknownHostException e) {
+            logger.warn("{}", e.getMessage());
+        }
+    }
+
+    /*
+     * Build VNet Frame
+     */
+    private final ArrayList<Byte> buildVNetFrame(ArrayList<Byte> macacoFrame2, @Nullable String gatewayLanAddress,
+            byte iUserIndex, byte iNodeIndex) {
+        if (gatewayLanAddress != null) {
+            ArrayList<Byte> frame = new ArrayList<>();
+            InetAddress ip;
+            try {
+                ip = InetAddress.getByName(gatewayLanAddress);
+            } catch (UnknownHostException e) {
+                logger.error("{}", e.getMessage());
```

Comment:
Bindings should only log to error if something severe happened, like the detection of a bug in your code. This could be debug. See this link for a description of the log levels: https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
fixed all classes